src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double sideA;
+    private double sideB;

```

Comment:
Use meaningful names
```suggestion
    private double width;
    private double height;
```